Runigram.java

```diff
@@ -59,49 +76,107 @@ private static void print(Color c) {
 	// This function is used for debugging purposes.
 	// For example, to check that some image processing function works correctly,
 	// we can apply the function and then use this function to print the resulting image.
-	private static void print(Color[][] image) {
+	private static void print(Color[][] image) 
+	{
 		//// Replace this comment with your code
+		for (int i = 0; i < image.length; i++) 
+		{
+            for (int j = 0; j < image[0].length; j++) 
+			{
+				print(image[i][j]);
+			}
+			System.out.println();
+		}	
 	}
 	
 	/**
 	 * Returns an image which is the horizontally flipped version of the given image. 
 	 */
-	public static Color[][] flippedHorizontally(Color[][] image) {
-		//// Replace the following statement with your code
-		return null;
+	public static Color[][] flippedHorizontally(Color[][] image) 
+	{
+		int rows = image.length;
+		int cols = image[0].length;
+		Color[][] newimage = new Color[rows][cols];
+			for (int x = 0; x < rows; x++)
+			{
+				for (int y = 0; y < cols; y++)
+				{
+					newimage[x][y] = image[x][cols-1-y];
+			    }
+		    }

```

Comment:
Here again, why x and y instead of row and col?
And indentations problem is everywhere.